reviews/2020-Network Service Mesh-annual.md

```diff
@@ -0,0 +1,192 @@
+  Funded by sponsorship from:
+  * Cisco
+  * Anthem
+
+## Annual Review Questions
+
+1.  _Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats._
+
+    [Network Service Mesh Devstats](https://networkservicemesh.devstats.cncf.io/)
+    
+    Highlights:
+    *  [31 contributors](https://networkservicemesh.devstats.cncf.io/d/22/prs-authors-table?orgId=1&var-period_name=Last%20year&var-repogroup_name=All) - up from 18 at acceptance into CNCF
+    *  [18 Organizations](https://networkservicemesh.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=commits)
+       
+       Including:
+       *  Cisco
+       *  Ericsson
+       *  Intel
+       *  Red Hat
+       *  SUSE
+       *  VMWare
+    * [Commits](https://networkservicemesh.devstats.cncf.io/d/2/commits-repository-groups?orgId=1&from=now-1y&to=now&var-period=d7&var-repogroups=All)
+      have accelerated since the beginning of our migration to a more modular multi-repo model.
+    
+2.  _How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)_
+
+    Network Service Mesh has four top level MAINTAINERS:
+    * Andre Sobolov (Xored)
+    * Ed Warnicke (Cisco)
+    * Frederick Kautz (Doc.ai)
+    * Nikolay Nikolaev (Kong)
+    
+    Of note: Two of those maintainers have remained engaged through employer changes
+    * Frederick Kautz: Red Hat -> doc.ai
+    * Nikolay Nikolaev -> VMWare -> Kong
+    
+    In addition, as the Network Service Mesh has grown, it has recently begun adding sub-MAINTAINERS to specific repos in the organization:
+    * Alexandre Menezes (Red Hat) - [nsm-operator](https://github.com/networkservicemesh/nsm-operator)
+    * Denis Tingajkin (Xored) - [fanout](https://github.com/networkservicemesh/fanout)
+    
+3.  _What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are end users of your project, please do so. (Feel free to link to an existing ADOPTERS file if appropriate.)_
+
+    At least three companies are beginning to build this into next generation architecture:
+    * [Anthem](https://anthem.ai/)
+    * [Bell Canada](https://www.bell.ca/)
+    * [Doc.ai](https://doc.ai/)
+ 
+    The project is also used in the [CNF Testbed](https://github.com/cncf/cnf-testbed).
+
+4. _How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)_
+
+   This is Network Service Mesh's first annual review, so we have no previously recorded goals from past annual reviews.
+
+5. _What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?_
```

Comment:
For documentation, there are several items marked as TODO in the https://github.com/networkservicemesh/networkservicemesh/blob/master/docs/README.md (linked from the main README page). Is there a plan to address them?

Reply:
We track our issues here: https://github.com/orgs/networkservicemesh/projects/1 and run a community meeting at 7:30 AM PT every Tuesday to track on

We'll cross-reference the TODO list here with what is in that project board and make sure everything is present.

Thanks for catching this!

Reply:
@fkautz the TODOs in the referenced README is more about technical documentation/demonstrations that is tbd. Copy/pasting here for the history:

* TODO - High Level Concepts
* TODO - Detailed dives into individual topics
* TODO - Adding a new interface
* TODO - VPN Gateway